src/Components/CreateImageWizard/formComponents/ReviewStepTextLists.js

```diff
@@ -0,0 +1,514 @@
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvAzureList = () => {
+  const { getState } = useFormApi();
+  return (
+    <TextContent>
+      <Text component={TextVariants.h3}>Microsoft Azure</Text>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Red Hat hosted image
+          <br />
+          <ExpirationWarning />
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Azure Tenant ID
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['azure-tenant-id']}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Subscription ID
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['azure-subscription-id']}
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dt}>
+          Resource group
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          {getState()?.values?.['azure-resource-group']}
+        </TextListItem>
+      </TextList>
+      <br />
+    </TextContent>
+  );
+};
+
+export const TargetEnvOtherList = () => {
+  return (
+    <>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Image type
+        </TextListItem>
+        <TextListItem component={TextListItemVariants.dd}>
+          Built image will be available for download
+        </TextListItem>
+      </TextList>
+      <br />
+    </>
+  );
+};
+
+export const FSCList = ({ ...props }) => {
+  const { getState } = useFormApi();
+  return (
+    <TextContent>
+      <TextList component={TextListVariants.dl}>
+        <TextListItem
+          component={TextListItemVariants.dt}
+          className="pf-u-min-width"
+        >
+          Configuration type
+        </TextListItem>
+        <TextListItem
+          component={TextListItemVariants.dd}
+          data-testid="partitioning-auto-manual"
+        >
+          {getState()?.values?.['file-system-config-radio'] === 'manual'
+            ? 'Manual'
+            : 'Automatic'}
+          {getState()?.values?.['file-system-config-radio'] === 'manual' && (
+            <>
+              {' '}
+              <Popover
+                position="bottom"
+                headerContent="Partitions"
+                hasAutoWidth
+                minWidth="30rem"
+                bodyContent={
+                  <FSReviewTable
+                    fsc={getState().values['file-system-configuration']}
```

Comment:
Could we call getState().values['file-system-configuration'] inside of the `<FSReviewTable />` instead of passing it in as a prop?